events_scrap/events_scrap.py

```diff
@@ -207,31 +213,29 @@ def process_event_doc(event_data, cluster_bash_data):
 
 
 def handle_arguments():
-    parser = ArgumentParser(description="Elastify events")
-    parser.add_argument("--inventory-url", help="URL of remote inventory", type=str)
-    parser.add_argument("--offline-token", help="offline token", type=str)
-    parser.add_argument("-es", "--es_server", help="Elasticsearch server", type=str)
-    parser.add_argument("-eu", "--es_user", help="Elasticsearch user", type=str)
-    parser.add_argument("-ep", "--es_pass", help="Elasticsearch password", type=str)
-    parser.add_argument("--index", help="Index", type=str)
-    parser.add_argument("--backup-destination", help="Path to save backup, if empty no back up saved",
-                        default=None, type=str)
-
-    return parser.parse_args()
+    return {
+        "inventory_url": utils.get_env("INVENTORY_URL", False),
+        "offline_token": utils.get_env("OFFLINE_TOKEN", True),
+        "es_server": utils.get_env("ES_SERVER", True),
+        "es_user": utils.get_env("ES_USER", False),
+        "es_pass": utils.get_env("ES_PASS", False),
+        "index": utils.get_env("ES_INDEX", True),
+        "backup-destination": utils.get_env("BACK_DESTINATION", False),
```

Comment:
```suggestion
        "backup_destination": utils.get_env("BACKUP_DESTINATION", False),
```

Reply:
I think you still didn't handle that
in another place you referred to it as ``backup_destination``